setup.py

```diff
@@ -5,35 +5,37 @@
 
 import mosql
 
-import sys
-extra = {}
-if sys.version_info >= (3,):
-    extra['use_2to3'] = True
```

Comment:
You added this during the sprint last year IIRC. But the support was broken after 0.9.1.